### DISCUSSION TITLE
API Docs

### DISCUSSION THREAD

**Discussion by Terryroux** [requarks/wiki#6552] [Help / Questions] [answered]
## Questions 
Hello, 
I'm using wiki in order to write some informations for my team. 

I would like to use also wiki.js in order to write API docs 
As we can see on the screenshot 👇 , it is written `coming soon`. 
Do you have an idea about a date to deliver this feature ? 

Thank you in advance

Terence 

![image](https://github.com/requarks/wiki/assets/138600492/8a5bc471-3984-4978-a87c-8d5295f201c4)

**Comment by NGPixel** [comment, marked as answer]
Follow https://feedback.js.wiki/wiki/p/api-graphql-docs-editor
